src/lib/seam/components/ClimateSettingScheduleTable/ClimateSettingScheduleTable.tsx

```diff
@@ -9,28 +12,67 @@ import { EmptyPlaceholder } from 'lib/ui/Table/EmptyPlaceholder.js'
 import { TableBody } from 'lib/ui/Table/TableBody.js'
 import { TableHeader } from 'lib/ui/Table/TableHeader.js'
 import { TableTitle } from 'lib/ui/Table/TableTitle.js'
+import { SearchTextField } from 'lib/ui/TextField/SearchTextField.js'
 import { Caption } from 'lib/ui/typography/Caption.js'
 
 import { ClimateSettingScheduleRow } from './ClimateSettingScheduleRow.js'
 
 export interface ClimateSettingScheduleTableProps {
   deviceId: string
+  disableSearch?: boolean
+  climateSettingScheduleFilter?: (
+    climateSettingSchedule: ClimateSettingSchedule,
```

Comment:
cc @mikewuu @razor-x, wanted to make sure we're using this property consistently